hazelcast/src/main/java/com/hazelcast/internal/util/phonehome/BuildInfoCollector.java

```diff
@@ -41,6 +41,7 @@ public void forEachMetric(Node node, BiConsumer<PhoneHomeMetrics, String> metric
         metricsConsumer.accept(PhoneHomeMetrics.BUILD_VERSION, imdgInfo.getVersion());
         metricsConsumer.accept(PhoneHomeMetrics.JET_BUILD_VERSION,
                 jetInfo == null ? "" : jetInfo.getVersion());
+        metricsConsumer.accept(PhoneHomeMetrics.JAVA_CLASSPATH, System.getProperty("java.class.path"));
```

Comment:
Shouldn't the users be able to control whether they send the classpath info or not? There could be some sensitive info they don't intend to share with us in embedded setups.

Just checked the fixed issue and it says so:
> It should be part of the Phone Home system and the user will have the ability to opt-out.

Reply:
I have clarified this with @dbrimley and they will be able to opt out as before via switching off the phone home entirely.

Reply:
Ok, so we're all or nothing on this. I just have the concern that the lack of separate control for the classpath will force projects to turn phone homes off entirely. But anyway, the code is in line with the intention, so I'm giving my :+1: